test5-findpairs/src/main/java/me/sieric/findpairs/Main.java

```diff
@@ -0,0 +1,79 @@
+package me.sieric.findpairs;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.layout.ColumnConstraints;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.Priority;
+import javafx.scene.layout.RowConstraints;
+import javafx.stage.Stage;
+
+/**
+ * Application for playing "Find pairs" game
+ * Accepts one integer parameter as command line argument
+ * Given integer should be even number between 2 and 8 and
+ * it defines number of rows and columns on the field.
+ */
+public class Main extends Application {
+
+    /**
+     * Initializes the window and passes size of the filed to {@link Controller}
+     * @param args command line arguments should contain one even number between 2 and 8
+     */
+    public static void main(String[] args) {
+        launch(args);
+    }
+
+    /**
+     * Starts the application
+     * @param primaryStage application window
+     */
+    @Override
+    public void start(Stage primaryStage) {
+        Parameters args = getParameters();
+        int size = Integer.parseInt(args.getUnnamed().get(0));
```

Comment:
Caused by: java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0